test/dummy.go

```diff
@@ -40,6 +40,11 @@ func NewDummyDA() *DummyDA {
 
 var _ da.DA = &DummyDA{}
 
+// Config returns the max blob size in bytes.
+func (d *DummyDA) Config() (uint64, error) {
+	return 64 * 64 * 482, nil
```

Comment:
i would just use max uint64 here instead of some magic numbers.

Reply:
Actually, I think we should be able set this value in the dummy, for testing purposes.

Reply:
Agreed.

Reply:
Made a default with option to override. Otherwise would need to copy consts to the caller everywhere.